### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Patriciawa** [javanile/vtiger#19]
Hi,

I've created a bash script that will install Vtiger, db and nginx (you can view it at my profile). However to actually install Vtiger you have to manually go through a form. Your file (vtiger-install.php) has a http bot that will do that for you. Can I just copy and execute the php file ? Thanks for your great work!

Regards,

Patrick

**Comment by francescobianco** [comment]
@Patriciawa Yes, of course, do this, it is opensource, it is free, do your best with this...
